eclipse/sw360
# feat(UI): CLI file clean up assistant

> Please provide a summary of your changes here.
> * Which issue is this pull request belonging to and how is it solving it? (*#1365*)
> * Did you add or update any new dependencies that are required for your change? - No

Issue: closes: #1365 

**Refer** - #1365 

### How To Test?
> Upload CLI files to Release, with variations in 
>    - copyright - (Content, File Hash)
>    - License Data - (licenseSPDXId, licenseName, licenseText, SourceFileHash)
>    - Obligations - Titile, Text, Licenses 
>    - Verify - Proper attachments are greyed out eith corrent data.
>    ![image](https://user-images.githubusercontent.com/56534710/131668814-8c90c098-af9d-412f-9f11-f8914b3ee93c.png)
 
> Test Attachment Usage/ Attachment Select in Generate Clearing Report/Readme OSS/Source Code Download of Project.
    ```
in project clearing / attachment selection / attachment usage views: this CLI file does not appear anymore
exception only for the projects where it has been actively selected (attachment usage has been set for tis file) - greyed out
    ```

> Have you implemented any additional tests? - No

### Checklist
Must:

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>
